htdocs/commande/class/commande.class.php

```diff
@@ -1113,6 +1113,8 @@ function createFromClone($socid=0)
 
 			if (! $error)
 			{
+				// force load new object before launch hook 
+				$this->fetch($this->id);
```

Comment:
Loading here is useless because code before already need that object is loaded.
A test at begin of method
if (empty($this->id)) return 'Error';